andela/AndelaEatsUI
# WIP #161477168 Add configuration file for code climate

#### What does this PR do?
- Adds configuration file for `codeclimate`

#### Description of Task to be completed?
- Add `.codeclimate.yml` file

#### How should this be manually tested?
- Click the `details` link on your PR to checkout `codeclimate` issue to ensure that those handles in the config file are not flagged

#### What are the relevant pivotal tracker stories?
- [#161477168](https://www.pivotaltracker.com/story/show/161477168)
